<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDialog</class>
 <widget class="QDialog" name="ChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatDialog</string>
  </property>
  <widget class="QTextEdit" name="textView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="messageEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
